README.md

```diff
@@ -36,6 +36,9 @@ For more visibility into which tests are running, use the `SpecReporter`:
 
 `SPEC_REPORTER=1 bin/test`
 
+By default the tests run with warnings disabled to reduce. To enable warnings, pass `VERBOSE=1`.
```

Comment:
I think you didn't finish that first sentence 😆

Reply:
Oops, updated.